modules/oadp-self-signed-certificate.adoc

```diff
@@ -80,9 +81,18 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
 ----
+
+.To fetch the backup logs, run the following command:
```

Comment:
This needs a space to be treated as a step. It also renders incorrectly in your build preview. 
![image](https://github.com/openshift/openshift-docs/assets/46728411/242e83ee-fe77-4fdc-acf7-03ae69e127e8)

Reply:
Updated and squashed